fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/api/ContainmentIndex.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.api;
+
+import java.util.stream.Stream;
+
+import org.fcrepo.kernel.api.models.FedoraResource;
+
+/**
+ * An interface defining the encapsulates containment relationships.
```

Comment:
The "encapsulates containment relationships" is kind of confusing.    Should it be `encapsulated`?  Or is this class a helper to determine what the containment relationships are for existing objects?

Reply:
will fix.